examples/src/bin/systimer.rs

```diff
@@ -51,21 +53,9 @@ fn main() -> ! {
         ALARM2.borrow_ref_mut(cs).replace(alarm2);
     });
 
-    interrupt::enable(
-        peripherals::Interrupt::SYSTIMER_TARGET0,
-        Priority::Priority1,
-    )
-    .unwrap();
-    interrupt::enable(
-        peripherals::Interrupt::SYSTIMER_TARGET1,
-        Priority::Priority2,
-    )
-    .unwrap();
-    interrupt::enable(
-        peripherals::Interrupt::SYSTIMER_TARGET2,
-        Priority::Priority3,
-    )
-    .unwrap();
+    interrupt::enable(Interrupt::SYSTIMER_TARGET0, Priority::Priority1).unwrap();
+    interrupt::enable(Interrupt::SYSTIMER_TARGET1, Priority::Priority2).unwrap();
```

Comment:
This seems to panic on the ESP32-S2, changing `SYSTEM_TARGET1`'s priority to `Priority3` seems to fix it. @bjoernQ @MabezDev is this okay to do?